scripts/babel/dev-expression-with-error-map/MapBuilder.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var invertObject = require('./invertObject');
+
+/*:: type ErrorMap = {  [id: string]: string; }; */
```

Comment:
Why are you using the comment version of flow types instead of using the js version?